contracts/Uniswap/UniswapAnchoredView.sol

```diff
@@ -12,69 +12,86 @@ struct PriceData {
     bool failoverActive;
 }
 
-contract UniswapAnchoredView is AggregatorValidatorInterface, UniswapConfig, Ownable {
+contract UniswapAnchoredView is
+    AggregatorValidatorInterface,
+    UniswapConfig,
+    Ownable
+{
     /// @notice The number of wei in 1 ETH
-    uint public constant ethBaseUnit = 1e18;
+    uint256 public constant ETH_BASE_UNIT = 1e18;
 
     /// @notice A common scaling factor to maintain precision
-    uint public constant expScale = 1e18;
+    uint256 public constant EXP_SCALE = 1e18;
 
     /// @notice The highest ratio of the new price to the anchor price that will still trigger the price to be updated
-    uint public immutable upperBoundAnchorRatio;
+    uint256 public immutable upperBoundAnchorRatio;
 
     /// @notice The lowest ratio of the new price to the anchor price that will still trigger the price to be updated
-    uint public immutable lowerBoundAnchorRatio;
+    uint256 public immutable lowerBoundAnchorRatio;
 
-    /// @notice The minimum amount of time in seconds required for the old uniswap price accumulator to be replaced
+    /// @notice The time interval to search for TWAPs when calling the Uniswap V3 observe function
     uint32 public immutable anchorPeriod;
 
     /// @notice Official prices by symbol hash
     mapping(bytes32 => PriceData) public prices;
 
     /// @notice The event emitted when new prices are posted but the stored price is not updated due to the anchor
-    event PriceGuarded(bytes32 indexed symbolHash, uint reporter, uint anchor);
+    event PriceGuarded(
+        bytes32 indexed symbolHash,
+        uint256 reporterPrice,
+        uint256 anchorPrice
+    );
 
     /// @notice The event emitted when the stored price is updated
-    event PriceUpdated(bytes32 indexed symbolHash, uint price);
+    event PriceUpdated(bytes32 indexed symbolHash, uint256 price);
 
     /// @notice The event emitted when failover is activated
     event FailoverActivated(bytes32 indexed symbolHash);
 
     /// @notice The event emitted when failover is deactivated
     event FailoverDeactivated(bytes32 indexed symbolHash);
 
-    bytes32 constant ethHash = keccak256(abi.encodePacked("ETH"));
+    bytes32 internal constant ethHash = keccak256("ETH");
```

Comment:
Following the N-02 suggestion, could we name this (and all references to this) `ETH_HASH` instead?